### PR TITLE
Task

### PR DESCRIPTION
**Pull request by amiraelsayedbadwy**
1. 
2. - Only store the score in preferences storage, if it is higher than the previous one, and we will show that they broke their record, on the screen.
3. -record data to the assigned leaderboard POST API for the difficulty
4. -the top notification will show for 5 seconds
5. 
6.  When the user ranked top ten, show this screen
7. we will make api call get ranks api  and if the average is less the average of current number one then display this message
8.  Make sure you are able to figure out the right rank that you placed
9. Add animation for wining

### Testing
android>home>start>easy>ready>launch>win page
ios>home>start>easy>ready>launch>win page

### CODE REVIEW

File: IslandLanding/IslandLanding/Views/WinPage.xaml [hunk 1]
```diff
@@ -30,7 +40,7 @@
         <Label.FormattedText>
           <FormattedString>
             <Span Text="{Binding UserTag}"/>
-            <Span Text=",  You have successfully landed on the island!"/>
+            <Span Text=" ,You have successfully landed on the island!"/>
```
Comment (saamerm) [inline IslandLanding/IslandLanding/Views/WinPage.xaml:43]: @amiraelsayedbadwy  did you change this?

**Review by saamerm — CHANGES_REQUESTED** [review]

File: IslandLanding/IslandLanding/Views/WinPage.xaml (hunk 1 above)
Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/Views/WinPage.xaml:43]: yes

Comment (saamerm) [reply IslandLanding/IslandLanding/Views/WinPage.xaml:43]: Please make this ", you have ...

Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/Views/WinPage.xaml]: done

**Review by saamerm — APPROVED** [review]

**Pull request merged**